### DISCUSSION TITLE
Github 2FA plus saml/sso and 2FA

### DISCUSSION THREAD

**Discussion by abcondas** [community/community#23688] [New to GitHub]
<p>My company just added a new github repo behind an okta identity.  To access this repo I have to sign-in to github as normal (with 2FA) and then additionally sign into okta (with 2FA).  I updated my personal access token to ‘Enable SSO’ and then hit the authorize button.  This allowed me to clone and fetch from the repo.  But it won’t let me push.  I receive a ‘repo not found’ error.  This occurs in all versions of git: git bash, git on windows, git ubuntu, jetbrains intellij, etc.  Any ideas?  Thanks!</p>

**Comment by ernest-phillips** [comment, marked as answer]
<p><a class="mention" href="/abcondas">@abcondas</a> :wave:</p>
<p>Welcome!</p>
<p>Have you checked with an admin of organization’s repository to see if you have write permissions?</p>

**Comment by abcondas** [comment]
<p>Yes, that was the problem.  I meant to update this thread with that info.  Sorry about that.</p>

**Comment by ernest-phillips** [comment]
<p>No worries! Glad You were able to get that resolved.</p>
